de-lib/src/main/java/org/iplantc/de/notifications/client/utils/NotificationUtil.java

```diff
@@ -129,13 +129,18 @@ public NotificationMessage getMessage(Notification notification,
                         AutoBeanCodex.decode(notFactory, PayloadData.class, payload).as();
                 String dataAction = dataPayload.getAction();
 
-                if ("file_uploaded".equals(dataAction)) {
+                if ("file_uploaded".equals(dataAction) || "UPLOAD_COMPLETE".equals(dataAction)) {
```

Comment:
Why is just this one uppercase? Not that completed uploads are part of the async tasks stuff really, but I'm curious, heh

Reply:
Good question!  I tried to search for where this payload action is set, but I can't find it.  But basically when I clicked on a file upload notification, the Data window wasn't opening like I expected, and then I noticed the actual action in the payload was `UPLOAD_COMPLETE`. Not sure if that means `file_uploaded` is no longer used or if that's an action for something else, so I kept them both for now.